rskj-core/src/test/java/org/ethereum/crypto/signature/Secp256K1ServiceNativeTest.java

```diff
@@ -0,0 +1,8 @@
+package org.ethereum.crypto.signature;
+
+public class Secp256K1ServiceNativeTest extends Secp256k1ServiceTest {
+
+    public Secp256K1ServiceNativeTest() {
+        super(new Secp256K1ServiceNative());
```

Comment:
Are we sure that this test will pass in any development environment? ie Windows 10

Reply:
For now yes. But when we actually implement this, we should consider OS. So, its a really good question for the next PR.